src/hapi_schema/db_org.py

```diff
@@ -25,13 +25,10 @@ class DBOrg(Base):
             "(reference_period_end >= reference_period_start) OR (reference_period_start IS NULL)",
             name="reference_period",
         ),
-        CheckConstraint(
-            "(hapi_replaced_date IS NULL) OR (hapi_replaced_date >= hapi_updated_date)",
-            name="hapi_dates",
-        ),
+        UniqueConstraint("acronym", "name"),
```

Comment:
This would mean that the combination of org name and acronym are unique, so orgs could only have one org type and reference period, right?  I haven't run into orgs with the same name and acronym but different types, so this is fine for me.

Reply:
Reference period is not included in this constraint - but it sounds like I should probably add it?